app/controllers/orders_controller.rb

```diff
@@ -2,4 +2,8 @@ class OrdersController < ApplicationController
 	def index
 		@orders = Order.all
 	end
+
+	def show
+		@order = Order.find(params[:id])
+	end
```

Comment:
indentation in this file seems to be weird. Most Ruby developers use 2 spaces indentation per level